app/assets/stylesheets/main.scss

```diff
@@ -56,3 +56,8 @@
   #summary-link {
     float: right;
   }
+
+  .structural_changes {
```

Comment:
Trivial: it looks like the other classes defined above use hyphens instead of underscores to separate words.

Reply:
Typically I've followed a hyphen for IDs, underscores for classes style (loosely) but happy to match the above.